src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -13,14 +13,13 @@ import { useDataArray } from '../../dataset-visualizer/VisProvider';
 
 function HeatmapVis(): JSX.Element {
   const dataArray = useDataArray();
-  const [rows, cols] = dataArray.shape;
-
   const [keepAspectRatio, showGrid] = useHeatmapConfig(
     (state) => [state.keepAspectRatio, state.showGrid],
     shallow
   );
 
   // width / height <=> cols / rows
+  const [rows, cols] = useDims();
```

Comment:
Just making use of the Heatmap's `useDims` hook

Reply:
I wonder if this hook is still relevant as you can get the same result from `useDataArray` and `dataArray.shape`. 

On the other hand, we could give it meaning by returning an object of type `{cols, rows}` so that it avoids switching the two when getting them through the hook.

Reply:
Yeah, it's still good for typing, I think. Should we also change the type `Dims`, then?